<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWin</class>
 <widget class="QMainWindow" name="MainWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1030</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Path to pyuic5.bat </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_pathToBat">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>641</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_browseBat">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>40</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Directory to process:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_pathToDirectoryToProcess">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>100</y>
      <width>651</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_browseDirProcess">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>100</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate code</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_output">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>951</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1030</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_browseDirProcess</sender>
   <signal>clicked()</signal>
   <receiver>MainWin</receiver>
   <slot>browse_directory_to_process()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>915</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>969</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_browseBat</sender>
   <signal>clicked()</signal>
   <receiver>MainWin</receiver>
   <slot>browse_path_to_bat()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>903</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>995</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>MainWin</receiver>
   <slot>gen_code()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>192</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>browse_directory_to_process()</slot>
  <slot>browse_path_to_bat()</slot>
  <slot>gen_code()</slot>
 </slots>
</ui>
